general.py

```diff
@@ -1,22 +1,23 @@
 ''' Some utility functions '''
 
 from bs4 import BeautifulSoup
-from urlparse import urljoin
+from urllib.parse import urlparse
```

Comment:
Can you undo this change, please?
The project is intended to be used with python 2.7 and python 2.7 doesn't have a `urllib.parse` package.